### ISSUE TITLE
ATK - test chip ram banks as seen by custom chips ?

### ISSUE THREAD

**Issue by penfold42** [keirf/Amiga-Stuff#15]
Another great piece of software from you - have you ever released anything crap ?!!

I’ve been working on resurrecting an old trapdoor expansion with a failed (unpublished) GAL.

I’ve obviously been making heavy use of ATK as part of this.

What I can’t 100% test is the chip ram banks as seen from the Agnus Denise and Paula.

Could you add video tests in each of the chip ram banks  based on what’s detected ?
Ie, an image at $00xxxx, $08xxxx, $10xxxx, $18xxxx

For bonus points borrow DiagROM scroll to test the blotter.

**Comment by keirf** [comment]
Do you suspect that the ram behaves differently when accessed by custom chips then? All accesses are mediated via Gary right?

**Comment by penfold42** [comment]
Yep - this design uses CAS bank switching switching and there are a couple of jumpers for mapping options.

It’s first bank can live at $08xxxx and would become the second half of chip ram on the old 500.

If I screw up my logic, bad stuff could happen from Agnus side while looking good from the CPU
